src/intent_pilot/run/app.py

```diff
@@ -56,13 +79,21 @@ def main():
             )
             break
         try:
-            operations = call_gpt_4_vision_preview_labeled(
-                client, messages, skip_som_draw_labels=["text"]
-            )  # skip labels some
+            if config.model == Models.GPTV4:
+                operations = call_gpt_4_vision_preview_labeled(
+                    client, messages, skip_some_draw_labels=["text"]
```

Comment:
we are using som = [set-of-mark](https://arxiv.org/abs/2310.11441) labels
```suggestion
                    client, messages, skip_som_draw_labels=["text"]
```

Reply:
My bad. Forgot to change it back once I realized this is some acronym I did not understand 😉